src/tests/utils/ApiTestSteps.ts

```diff
@@ -172,11 +172,11 @@ export async function getYotiSessionsConfiguration(sessionId: any): Promise<any>
 	}
 }
 
-export async function putYotiSessionsInstructions(sessionId: any): Promise<any> {
+export async function putYotiSessionsInstructions(sessionId: any, payload: any): Promise<any> {
```

Comment:
Please can we have some types for these parameters?

Reply:
Payload is JSON, believe `sessionId` is implicitly assigned by JEST deconstructing the array passed into `it.each` as `string | number` - would this be preferable to `any`